testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -725,6 +725,37 @@ protected static SortedSet<String> keywords(@Nullable String dialect) {
             + ".*");
   }
 
+  /** Test case for <a href="https://issues.apache.org/jira/browse/CALCITE-5997">[CALCITE-5997]</a>
```

Comment:
![image](https://github.com/apache/calcite/assets/45968640/bbac3362-ba68-406e-b5f5-4a44e860312d)

IMO, a tag should contain Jira summary.